### ISSUE TITLE
Correct command to open URL generated, but doesn't open from inside `NVIM v0.7.0-dev`

### ISSUE THREAD

**Issue by alichtman** [ruanyl/vim-gh-line#55]
When I trigger the plugin, I see

```
vim-gh-line executing: xdg-open https://github.com/alichtman/shallow-backup/blob/a5b36f3532f07c4fc577495a2d476f8e133456ff/README.md#L239
```

but the page never opens in Firefox. Running that command manually in a terminal *does* open the page.

**Comment by zodman** [comment]
@alichtman  check prev message!

**Comment by alichtman** [comment]
@zodman Yup, I agree with your analysis (and thumbs'd up your comment the day you posted it 😃)

Leaving the issue open, as there's no proper fix

**Comment by zodman** [comment]
@sebastian-palma PR please please!

**Comment by alichtman** [comment]
I found out that [`vim-fugitive`](https://github.com/tpope/vim-fugitive) has this functionality baked in, so I added these two remappings and dropped the `vim-gh-line` plugin from my config.

```
" Open selection on github
nnoremap go :.GBrowse<CR>
vnoremap go :'<,'>.GBrowse<CR>
```

**Comment by sebastian-palma** [comment]
@zodman see [this](https://git.sr.ht/~sebastian-palma/vim-gh-line/commit/4c9c4be2b78749290bfc4eed57b9bcf728b75de3), this is the only change I needed to make it work.
